### PR TITLE
Add route_table

### PR DESCRIPTION
**Pull request by k1LoW**
Add command `rt` for aws_route_table

### CODE REVIEW

File: spec/lib/terraforming/resource/route_table_spec.rb [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                                                                      "id"=>"rtb-efgh5678",
+                                                                      "attributes" => {
+                                                                        "id" => "rtb-efgh5678",
+                                                                        "route.#" => "1",
+                                                                        "tags.#" => "1",
+                                                                        "vpc_id" => "vpc-ab123cde"
+                                                                      }
+                                                                    }
+                                                                  }
+                                                                })
```
Comment (dtan4) [inline spec/lib/terraforming/resource/route_table_spec.rb]: Please indent these lines as below:

```ruby
expect(described_class.tfstate(client: client)).to eq({
  "aws_route_table.my-route-table" => {
    "type" => "aws_route_table",
    "primary" => {
      "id" => "rtb-a12bcd34",
      "attributes" => {
        "id" => "rtb-a12bcd34",
        "route.#" => "4",
        "tags.#" => "1",
        "vpc_id" => "vpc-ab123cde",
      }
    }
  },
  "aws_route_table.my-route-table-2" => {
    "type" => "aws_route_table",
    "primary" => {
      "id"=>"rtb-efgh5678",
      "attributes" => {
        "id" => "rtb-efgh5678",
        "route.#" => "1",
        "tags.#" => "1",
        "vpc_id" => "vpc-ab123cde"
      }
    }
  }
})
```

Comment (k1LoW) [inline spec/lib/terraforming/resource/route_table_spec.rb]: fixed!!

**Comment by dtan4** [comment]
LGTM :+1: Thanks :exclamation:

**Comment by k1LoW** [comment]
:star:

**Comment by dtan4** [comment]
Released as v0.5.0 :shipit:

**Pull request merged**